crates/oxc_minifier/src/ast_passes/peephole_replace_known_methods.rs

```diff
@@ -33,88 +33,92 @@ impl PeepholeReplaceKnownMethods {
         Self { changed: false }
     }
 
+    fn get_string_literal<'a>(expr: &Expression<'a>) -> Option<&'a str> {
+        match expr {
+            Expression::StringLiteral(lit) => Some(lit.value.as_str()),
+            Expression::TemplateLiteral(lit) if lit.is_no_substitution_template() => {
+                lit.quasi().map(|s| s.as_str())
+            }
+            _ => None,
+        }
+    }
```

Comment:
i'm not sure whether it's worth having this here?
this would get removed by #6486 in multiple passes
6486 also work with some additional cases

Reply:
Yes, I just saw this PR, so I made some improvements.

Reply:
Should we add a `get_string_literal` to `NodeUtil` to retrieve string literals?

Reply:
yeah actually i just had that thought also
using `get_string_literal` will catch more cases.
do you mind adding additional tests for these please?

Reply:
Sure